lib/persistent.js

```diff
@@ -31,6 +35,53 @@ module.exports = class PersistentNode {
     return this.dht.id.equals(TMP)
   }
 
+  onget (req, { mutable }) {
+    if (!req.target) return req.reply(null)
+    const store = mutable ? this.mutables : this.immutables
+    if (mutable) {
+      const k = req.target.toString('hex')
+      const value = store.get(k)
+      if (!value) return req.reply(Buffer.from('')) // empty response so initial puts can still find nodes
```

Comment:
empty value not required, just reply(null)

Reply:
I did do this and it meant that I couldn't get any nodes for an initial put, I'll try it again, perhaps it was another reason